onnxruntime/test/ir/graph_test.cc

```diff
@@ -98,8 +100,39 @@ static bool RegisterCustomSchemas() {
         fail_shape_inference("try harder");
       });
 
+  // Fake Op where the domain for the Op itself and the Ops in the function body is not same.
```

Comment:
> itself and the Ops in the function body is not same [](start = 40, length = 52)

Would be nice to document which is which for readability.

Reply:
add more comments + changed the test to include ops from 3 different domain. Some ops are fake and some are not. The point of this test is to make sure when function op and the function body ops don't belong to the same domain the function initialization still works and nodes are mapped to correct domain + version.